contents/blog/product-market-fit-game.md

```diff
@@ -0,0 +1,351 @@
+ 
+* **Your product isn't solving your users' problem**, hence there is no point coming back. Build a new product for the same problem, or fix your existing product. Restart level 4.
+
+* **Your product is painful to use.** Get feedback from users, look at session replays, or watch people trying to use it.
+
+* **Your users don't need to come back repeatedly.** Perhaps you've automatically solved the problem with no further user work. Proceed to Level 5 and validate this by seeing if they will pay for it. Don't force product engagement if it's irrelevant to solving the problem.
+
+* **Users use your product but don't encounter your problem often.** Proceed to Level 5. It may be hard to get significant revenue for an infrequent problem, or it may be that you don't experience much demand. If the number of users that experience this problem is huge even if the order value is tiny, that's ok. If it was extremely hard finding the first users _and_ it's hard to charge much for the product, this problem isn't a big enough one – go back to level 1.
+
+### Level 5 - Onboard your first 5 reference paying customers
+
+Start by figuring out your [Ideal Customer Profile](/blog/creating-ideal-customer-profile).
+
+Write down needs, haves and attributes you think your Ideal Customer Profile users might have in common. These should be _very_ specific. They could be individual-level and/or company-level things.
+
+Do not just assume one or two things like "engineers in startups with over 100 staff". Definitely do not just list an industry. The specific things we considered at PostHog were:
+
+![psothog icp](../images/blog/pmf-game-guide/posthog-icp.png)
+
+Feel free to include some anti-needs and haves too – i.e. things your Ideal Customer Profile _doesn't_ need or have.
+
+List out all the potential users you could sell down the first column in a spreadsheet, and score them against all of these attributes. Color-code the spreadsheet so you can spot patterns easily. As you start to see if people are happy to pay 
+
+Be upfront with customers. This step may prove tricky if you never set any commercial expectation with them early on. If so, you may find it easier to sell to newer users with clearer expectations. Use your website to make clear that your service is a paid one, or will be once you're of beta. 
+
+You will also need some pricing. You are likely to change this a bunch of times. Step 1 is to create some kind of internal table for you to use and iterate. Step 2 may be to make this transparent online and step 3 - for companies focusing on self-serve - is to enable people to pay without talking to you. Don't leap straight to self-serve though, since you'll get fewer lessons from users – you're not trying to optimize revenue growth early on, you're trying to optimize your ability to learn.
```

Comment:
```suggestion
You will also need pricing. You are likely to change this many times. 
1. Create some kind of internal table for you to use and iterate. 
2. Potentially make this transparent online.
3. For companies focusing on self-serve - is to enable people to pay without talking to you. Don't leap straight to self-serve though, since you'll get fewer lessons from users – you're not trying to optimize revenue growth early on, you're trying to optimize your ability to learn.
```